RELEASES.md

```diff
@@ -57,6 +58,110 @@ Get acquainted with the latest EDP releases.
+
+Our platform now integrates with Large Language Models, such as ChatGPT, as an intelligent [chat assistant](https://docs.kuberocketci.io/docs/next/user-guide/add-ai-assistant). Depending on your specified settings, it can perform a variety of tasks. We can also set up this assistant for you. Please contact our team for more details.
+
+A new release brings a new look. Check out our new [documentation site](https://docs.kuberocketci.io/), which features improved navigation and documentation versioning support.
+
+We have updated terminology within the platform. Environments are now referred to as Deployment Flows, and Stages are now Environments. These changes align our terminology with globally recognized terms, reducing user confusion. For more details, please refer to our updated [Glossary](https://docs.kuberocketci.io/docs/next/glossary).
+
+We have added the ability to specify custom build and review pipelines with just one click when editing or creating new codebase branches. This enhancement increases flexibility when working with codebases, allowing for easier modifications to pipeline logic based on specific requirements.
+
+Another significant enhancement is the introduction of a new section called Pipelines. This section centralizes pipeline observability by consolidating all pipelines in one place. We are continuing to migrate Tekton functionality into our platform. Consequently, we have consolidated our pipelines into a single location. The new Pipelines section displays the entire Tekton Pipeline Run history, simplifying pipeline management and observability.
+
+Users can now deliver their applications in the form of RPM packages, implemented using Ansible. Correspondingly, support for C and C++ languages with make and CMake build tools has been added to the platform.
+
+We have also introduced the ability to [customize Environment deletion](https://docs.kuberocketci.io/docs/next/operator-guide/cd/customize-environment-deletion). This feature is particularly useful for users with applications that have specific configurations or dependencies, e.g. database cleanup, infrastructure tearing down, etc. Customized clean flows make it easier to maintain components throughout their lifecycle.
+
+Additionally, we have added new functionality to our deployment pipelines. A new tab called Variables has been added to the Environment (formerly Stage) details page. In this tab, users can specify key-value parameters, which will be transmitted within the application as environment variables. This improvement allows users to configure application deployment settings without leaving the KubeRocketCI portal.
+
+Deploy pipelines now support manual approval. When creating a custom deploy pipeline, users can include a checkpoint to prompt for approval or rejection before proceeding to the next step.
```

Comment:
Deployment pipelines have been enhanced with a manual approval feature. This allows developers to integrate a checkpoint within their custom deployment pipelines, requiring a manual review and approval process before the pipeline can advance to the next stage. This ensures greater control and oversight over the deployment process.